client_integration_test.go

```diff
@@ -2463,11 +2473,11 @@ func benchmarkWriteTo(b *testing.B, bufsize int, delay time.Duration) {
 	f.Write(data)
 	f.Close()
 
+	buf := new(bytes.Buffer)
```

Comment:
bytes.NewBuffer(make([]byte, 0, size)) would remove some variance, as the first run no longer needs to allocate.

Reply:
Indeed, I thought of that. I guess since I’m fixing small details elsewhere I should cover this one too, right? 😆 

👍 on keeping me on the straight and narrow!